scripts/make-bundle

```diff
@@ -79,17 +83,33 @@ def create_scratch_dir():
 def download_package_tarballs(dirname, packages):
     with cd(dirname):
         for package, package_version in packages:
-            run('%s -m pip download %s==%s %s' % (
-                sys.executable, package, package_version, PIP_DOWNLOAD_ARGS
-            ))
+            run(
+                f'{sys.executable} -m pip download {package}=={package_version}'
+                f' {PIP_DOWNLOAD_ARGS}'
+            )
+
+
+def download_package_wheels(dirname, packages):
+    with cd(dirname):
+        for package, package_version in packages:
+            run(
+                f'{sys.executable} -m pip download {package}=={package_version}'
+                f' --only-binary :all:'
+            )
+
+
+def validate_that_wheels_are_universal(dirname):
```

Comment:
non-blocking: I don't think it's required that the wheels are universal, only that wheels are available for the platform the user is installing AWS CLI on. It doesn't look like this will be an issue anytime soon and if it is, it'll be caught the by CI so I'm fine with this.